packages/MSBot/docs/update-service.md

```diff
@@ -0,0 +1,41 @@
+## Updating Service
+To update a service with new information you can use the msbot update set of commands
+
+```shell
+msbot update [serviceType] --[requiredArg] [value]
+```
+
+This allows you to rename a service, update keys, etc.
+
+| ServiceType | Description                          | Required argument |
+|-------------|--------------------------------------|-------------------|
+| appinsights | update Azure AppInsights             | --serviceName     |
+| blob        | update Azure Blob storage            | --serviceName     |
+| cosmosdb    | update Azure CosmosDB                | --serviceName     |
+| dispatch    | update Dispatch model                | --applicationId   |
+| endpoint    | update endpoint                      | --endpoint        |
+| generic     | update generic service configuration | --url             |
+| luis        | update LUIS application              | --applicationId   |
+| qna         | update QNA service                   | --kbId            |
+
+These commands take the same args as the msbot connect <servicetype> commands, but only some properties can be updated.
```

Comment:
It would be useful to add the list of properties that can't be updated.

Reply:
Yes. An example list of properties is listed right after this line.